src/compiler/checker.ts

```diff
@@ -16856,6 +16856,8 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
                 const propType = getTypeOfSymbol(prop);
                 return accessExpression && getAssignmentTargetKind(accessExpression) !== AssignmentKind.Definite ?
                     getFlowTypeOfReference(accessExpression, propType) :

```

Comment:
now that this has multiple entries, can you format it cond-style? Like so:

```ts
return cond1 ? cons2
    : cond2 ? cons2
    : default;
````

Reply:
done #prettierwhereareyou 😉